src/Service.js

```diff
@@ -71,11 +73,12 @@ class Service {
    * @return {Task} Returns the task object.
    */
   task(taskName) {
-    return new Task(this._server, this.name, taskName);
+    return new Task(this, taskName);
   }
 
   /**
    * Retrieves the array of task info objects available on the service.
+   * @version 1.1.0
```

Comment:
I added that to the docs since that method was added for 1.1.0.  That make sense?